### ISSUE TITLE
streamingAssetsUrl Missing in v7

### ISSUE THREAD

**Issue by NikitaVr** [jeffreylanters/react-unity-webgl#165]
The docs say setting the `streamingAssetsUrl` is supported since version 6, but I am using version 7 for Unity 2019, and I don't see any way to set streamingAssetsUrl

Defining the Streaming Assets URL https://github.com/elraccoone/react-unity-webgl#defining-the-streaming-assets-url

IUnityConfig doesn't seem to have a `streamingAssetsUrl` in version 7

```
export default interface IUnityConfig {
    /**
     * The unique identifier helps you getting the instance
     * of your context from any other class.
     * @type {string}
     */
    id?: string;
    /**
     * Unity Module injection.
     * @type {Object}
     */
    modules?: Object;
    /**
     * You can default your Unity Version. The library may
     * contain future patches for specific Unity versions.
     * It's not needed but is recommended to define a version.
     * @type {UnityVersion}
     */
    unityVersion?: UnityVersion;
    /**
     * Since the Unity canvas itself does not respond to the resizing
     * of it's container we have to manually do this. A width and height
     * of 100% does not seem to work, so we have to fetch it's parent's
     * size to adject the canvas.
     * @type {boolean}
     */
    adjustOnWindowResize?: boolean;
}
```

**Comment by jeffreylanters** [comment]
Hi @NikitaVr, I noticed you've closed the issue. Did you manage to get it to work?

**Comment by gamereat** [comment]
Hey @jeffreylanters, We're hitting a similar issue. Looks like it was added in at commit 88621627f21c382da43986a4a933311724a02d53 Do you know you plan to back port it to 7.X?

**Comment by gamereat** [comment]
Hey @jeffreylanters, Yeah. We are making use of addressable to improve performance which is held within streaming assets. Which is currently not possible with 7.x

**Comment by jeffreylanters** [comment]
I've been looking into this, if I remember this correctly the `streamingAssetsUrl` is provided via the `modules` parameter, which directly interfaces with Unity's WebGL build Module extension. Could you try to implement this? I think this should work already. Let me know!

```js
let unityContent = new UnityContent(
  "MyGame/Build.json",
  "MyGame/UnityLoader.js", {
    modules: {
      streamingAssetsUrl: "..."
    }
  }
);
```

**Comment by gamereat** [comment]
> I've been looking into this, if I remember this correctly the `streamingAssetsUrl` is provided via the `modules` parameter, which directly interfaces with Unity's WebGL build Module extension. Could you try to implement this? I think this should work already. Let me know!
> 
> ```js
> let unityContent = new UnityContent(
>   "MyGame/Build.json",
>   "MyGame/UnityLoader.js", {
>     modules: {
>       streamingAssetsUrl: "..."
>     }
>   }
> );
> ```

With above code I'm getting the following errors on latest 7.x
exception thrown: TypeError: Module.streamingAssetsUrl is not a function,TypeError: Module.streamingAssetsUrl is not a function

**Comment by jeffreylanters** [comment]
About to publish version 7.1.11 which includes support for Streaming Asset URL's. See [LTS-7.X/Feature/Streaming-Assets-URL-Support](https://github.com/jeffreylanters/react-unity-webgl/tree/lts-7.x/feature/streaming-assets-url-support).

**Comment by jeffreylanters** [comment]
I've released [7.1.11](https://www.npmjs.com/package/react-unity-webgl/v/7.1.11).

**Comment by jeffreylanters** [comment]
i've released version [7.1.12](https://github.com/jeffreylanters/react-unity-webgl/tree/7.1.12) adding support for Streaming Asset URLs and including updated documentation. If you have any more questions feel free to re-open this issue.

Thanks for opening an issue and best of luck with your project!

**Comment by gamereat** [comment]
That worked first time! Thanks for your help
